ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/MS_SDK_7.1/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+#######################################
+# Microsoft Windows SDK for Windows 7 #
+#######################################
+
+- name: Test if MS SDK is installed
+  win_stat:
+    path: 'C:\Program Files\Microsoft SDKs\v7.1'
+  register: ms_sdk_installed
+  tags: MS_SDK_7.1
+
+- name: Test if MS SDK is already downloaded
+  win_stat:
+    path: 'C:\temp\MS_SDK.zip'
+  register: ms_sdk_download
+  tags: MS_SDK_7.1
+
+- name: Download pre compiled freetype.zip
+  win_get_url:
+    url: https://ci.adoptopenjdk.net/userContent/MicrosoftSDKs.zip
```

Comment:
It would be good to document where this comes from originally.

Reply:
Comment needs to be changed as it says freetype :-)